tf2_ros/src/static_transform_broadcaster_program.cpp

```diff
@@ -40,87 +40,126 @@
 #define ROS_FATAL printf
 #define ROS_INFO printf
 
+//TODO(clalancette re-enable this)
+// bool validateXmlRpcTf(XmlRpc::XmlRpcValue tf_data) {
```

Comment:
I think this is the first place I've seen un-ported code commented out with `//` instead of block comments? But I feel like I've seen some discussion about this so it might be that this makes more sense for some reason.